.travis.yml

```diff
@@ -17,9 +17,9 @@ branches:
 matrix:
   include:
     - php: 5.5
-      env: SYMFONY_VERSION='2.3.*'
+      env: SYMFONY_VERSION='2.5.*' # Remove in July 2015
     - php: 5.5
-      env: SYMFONY_VERSION='2.5.*'
+      env: SYMFONY_VERSION='2.3.*' # LTS - Remove in May 2013
```

Comment:
As per http://symfony.com/doc/current/contributing/community/releases.html

Version|Feature Freeze|Release|End of Maintenance|End of Life
---|---|---|---|---
2.3|03/2013|05/2013|05/2016 (36 months)|05/2017